transcom/mymove
# B 19045 INT - Hide Request Diversion and Request Cancellation when Cancellation Requested

## [B-19045](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19045)

## Summary

Mostly does what it says on the tin.

Also, updates the tests for hiding "Request Cancellation" because that was already supposed to be hidden but the existing tests to verify that weren't picking up on it.

Also also, I made some capitalization and spelling more consistent. Apparently MilMove uses "Canceled" rather than "Cancelled." Turns out both are considered correct (one L in cancelation generally isn't), but the double L is more common in British English. I've always used two Ls.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Create a new HHG or identify one for which a cancellation request can be made (today must be before the actual pickup date, I deleted the pickup date from the DB for AMDORD-01 for most of my testing)
2. If creating a new HHG get it past SC approval.
3. Login as a TOO and navigate to the MTO page of the move you created or identified.
4. Verify you see both the Request Cancellation and Request Diversion buttons. 
5. Request a cancellation on the HHG shipment.
6. Verify you no longer see either the Request Cancellation or Request Diversion button.
7. Log into the Prime Simulator, locate the move, and cancel the shipment (from the update shipment page).
8. Log back in as a TOO, and navigate back to the MTO page for the same move.
9. Verify the move shows as canceled and that you still don't see either the Request Cancellation or Request Diversion button

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
